scss/elements/buttons.scss

```
@@ -73,3 +129,28 @@
     }
   }
 }
+
+// Btn dot
+.btn-extra-pixelize {
+  $border-size: 4px;
+
+  box-sizing: content-box;
+  display: inline-block;
+  padding: 6px 8px;
+  margin: $border-size;
+  cursor: pointer;
+  border: none;
+  @include btn-extra-pixelize-style($base-color, #fff, #e7e7e7, #adafbc);
+
+  $types:
+    "primary" #fff #92cc41 #76c442 #4aa52e, 
+    "success" #fff #209cee #108de0 #006bb3,
+    "warning" $base-color #f7d51d #f2c409 #e59400, 
+    "error" #fff #e76e55 #ce372b #8c2022;
+
+  @each $type in $types {
+    &.#{nth($type, 1)} {
```

please add the prefix of `is-` 🙏 

for example, `.btn.is-rounded.is-primary`, `.btn.is-rounded.is-success`...

```suggestion
    &.is-#{nth($type, 1)} {
```